kornia/enhance/adjust.py

```diff
@@ -529,6 +529,76 @@ def adjust_brightness_accumulative(image: Tensor, factor: Union[float, Tensor],
     return img_adjust
 
 
+def adjust_sigmoid(image: Tensor, cutoff: float = 0.5, gain: float = 10, inv: bool = False) -> Tensor:
+    """Adjust sigmoid correction on the input image tensor.
+
+    The input image is expected to be in the range of [0, 1].
+
+    Reference:
+    [1]: Gustav J. Braun, "Image Lightness Rescaling Using Sigmoidal Contrast Enhancement Functions",
+        http://markfairchild.org/PDFs/PAP07.pdf
+
+    Args:
+        image: Image to be adjusted in the shape of :math:`(*, H, W)`.
+        cutoff: The cutoff of sigmoid function.
+        gain: The multiplier of sigmoid function.
+        inv: If is set to True the function will return the inverse sigmoid correction.
+
+    Returns:
+         Adjusted tensor in the shape of :math:`(*, H, W)`.
+
+    Example:
+        >>> x = torch.ones(1, 1, 2, 2)
+        >>> adjust_sigmoid(x, gain=0)
+        tensor([[[[0.5000, 0.5000],
+                  [0.5000, 0.5000]]]])
+    """
+    KORNIA_CHECK_IS_TENSOR(image, "Expected shape (*, H, W)")
+
+    if inv:
+        img_adjust = (1 - 1 / (1 + torch.exp(gain * (cutoff - image))))
```

Comment:
```suggestion
        img_adjust = (1 - 1 / (1 + (gain * (cutoff - image)).exp()))
```
more pythinic toward supporting other frameworks

Reply:
Made more pythinic